config/verification-code.php

```diff
@@ -29,10 +29,10 @@
     |--------------------------------------------------------------------------
     |
     | A verification code is only valid for a certain amount of time. Here
-    | you define after how many hours a verification code will expire.
+    | you define after how many seconds a verification code will expire.
     |
     */
-    'expire_hours' => 1,
+    'expire_seconds' => 1 * 60 * 60,
```

Comment:
can you update the readme as well; the "config settings" docs still reference "expire hours"